packages/core/upload/admin/src/hooks/useFolderStructure.js

```diff
@@ -0,0 +1,42 @@
+import { useQuery } from 'react-query';
+
+import pluginId from '../pluginId';
+import { axiosInstance, getRequestUrl } from '../utils';
+
+const FIELD_MAPPING = {
+  name: 'label',
+  id: 'value',
+};
+
+const renameKeys = (obj, fn) =>
```

Comment:
What do you think of exporting this function so you can test it ?

Reply:
I'm testing the whole `useFolderStructure()` hook and thought it would be sufficient. But I don't mind extracting / unit testing it.

Reply:
Added in 65a5f2ed34bcdabcb60cd5b2cf3b47c3faaee782